src/pc/jerry/refresh.sh

```diff
@@ -1,6 +1,9 @@
 set -e
 
-cd ~/jerryscript_build
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+PROJECT_HOME=$SCRIPT_DIR/../../../..
+
```

Comment:
Can we please add ooooone more little change and ensure that the jerryscript build directory exists, and if it doesn't, state a clear human-readable warning saying that you should clone it etc?

One issue I observe is that the spade directions specifically tell you to check out the jerryscript_build repo into your home directory, but you may have checked out Spade somewhere else (for instance, I put stuff in ~/Work).  This causes this script to fail.